### PR TITLE
Styling team id

### PR DESCRIPTION
**Pull request by duanxianbo**

### CODE REVIEW

File: src/components/internal/Modals/Modal.shared.module.scss [hunk 1]
```diff
@@ -24,3 +24,21 @@
         }
     }
 }
+
```
Comment (naturetime10) [inline src/components/internal/Modals/Modal.shared.module.scss]:
```suggestion
```

File: src/components/internal/Modals/Modal.shared.module.scss [hunk 2]
```diff
@@ -24,3 +24,21 @@
...
+    font-weight: bold;
+    justify-content: center;
+    margin-top: 20px;
+    padding: 2px 0;
+    width: 120px;
+
+    .Id {
+        margin-left: 10px;
+    }
+}
```
Comment (naturetime10) [inline src/components/internal/Modals/Modal.shared.module.scss]: I think this should be put in a separate scss file before it's not really tied to modal.

**Review by naturetime10 — CHANGES_REQUESTED** [review]

File: src/components/internal/Modals/Modal.shared.module.scss (hunk 2 above)
Comment (duanxianbo) [reply src/components/internal/Modals/Modal.shared.module.scss]: maybe we should make a separate component?

Comment (naturetime10) [reply src/components/internal/Modals/Modal.shared.module.scss]: Importing shared scss is fine. I feel making a separate component for this case is a bit over killing before we don't have any logic inside.

Comment (duanxianbo) [reply src/components/internal/Modals/Modal.shared.module.scss]: okey, that is done

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (naturetime10) [inline .vscode/launch.json]: Do we need this file?

**Review by naturetime10 — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 3 above)
Comment (duanxianbo) [reply .vscode/launch.json]: no

**Review by naturetime10 — APPROVED** [review]

**Pull request merged**